app/api/dao/mentorship_relation.py

```diff
@@ -191,26 +179,21 @@ def reject_request(user_id, request_id):
         return {'message': 'Mentorship relation was rejected successfully.'}, 200
 
     @staticmethod
+    @email_verification_required
     def cancel_relation(user_id, relation_id):
-
-        user = UserModel.find_by_id(user_id)
-
-        # verify if user exists
-        if user is None:
-            return {'message': 'User does not exist.'}, 404
-
+        #user = UserModel.find_by_id(user_id)
```

Comment:
No code comments

Reply:
My mistake..... corrected in next commit